src/components/SlideViewer/Slide.js

```diff
@@ -1,15 +1,35 @@
 import React from "react";
 import styled from "styled-components";
 
-function Slide() {
-  return <SlideContainer>Slide</SlideContainer>;
+function Slide({ slideData }) {
+  const slideItems = slideData.items;
+  const textItems = slideItems.filter((item) => item.type === "text");
+  const imgItems = slideItems.filter((item) => item.type === "image");
+
+  return (
+    <SlideContainer>
+      {textItems.map((item) => {
+        return (
+          <TextWrapper key={item.id} props={item}>
+            {item.content.value}
+          </TextWrapper>
+        );
+      })}
```

Comment:
TextWrapper component는 하나의 표현식이므로, 가독성을 위해 다음과 같이 수정해주시면 더욱 깔끔해질 것 같습니다!

```suggestion
      {textItems.map((item) => (
        <TextWrapper key={item.id} props={item}>
          {item.content.value}
        </TextWrapper>
      ))}
```

Reply:
이게 아마 map을 쓰게 되면 return이 필수여서 return 을써야할꺼에요!